vorto-dashboard/index.js

```diff
@@ -56,9 +56,10 @@ apiRouter.get('/devices', (req, res) => {
 function checkSimulatorsRunning() {
   return new Promise((res, rej) => {
     exec("kill -0 $(ps aux | grep -E '[p]ython RaspberryPiTutorialApp_new.py|TraciApp.py' | grep -v grep | awk '{print $2}')",
```

Comment:
Please rename RaspberryPi to PMSMotorSimulator.py and TraciApp.py to TraciTagSimulator.py